middleware/faq.rst

```diff
@@ -466,6 +466,15 @@ The ``--pipeline-dot`` argument can also be used to create a version of this gra
 That ``...`` should be replaced by most of the arguments you'd pass to :any:`pipetask <lsst.ctrl.mpexec-script>` that describe *what* to run (which tasks, pipelines, configuration, etc.), but not the ones that describe how, or what to use as inputs (no collection options).
 See ``pipetask build --help`` for details.
 
+For a modern, web-friendly alternative, the ``--pipeline-mermaid`` argument can be used to generate the same graph in `Mermaid.js`_ format:
+
+.. code:: sh
+
+    $ pipetask build ... --pipeline-mermaid pipeline.mmd
+    $ mmdc -i pipeline.mmd -o pipeline.svg
+
+The ``mmdc`` command is part of the `Mermaid CLI`_, which can render Mermaid definition files into PNG, SVG, or PDF files. Alternatively, the resulting ``.mmd`` file can be rendered with tools like the `Mermaid Live Editor`_ or Markdown platforms supporting Mermaid syntax such as GitHub. This format is particularly useful for creating interactive, easily shareable pipeline graphs.
+
 This graph will often reveal some unexpected input dataset types, tasks, or relationships between the two that make it obvious what's wrong.
```

Comment:
I would change this line to start:
```
The visualized graph will often reveal...
```
This paragraph seems like it relates to the prior point alone (i.e., mermaid), but it's actually referencing any of the visualization methods described above.